packages/neuron-ui/src/components/BroadcastTransaction/index.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { useCallback, useMemo, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useNavigate } from 'react-router-dom'
+import { isSuccessResponse, RoutePath, isMainnet as isMainnetUtil, useDidMount, useGoBack, getExplorerUrl } from 'utils'
+import Dialog from 'widgets/Dialog'
+import AlertDialog from 'widgets/AlertDialog'
+import { useDispatch, useState as useGlobalState } from 'states'
+import {
+  broadcastTransactionOnly,
+  getCurrentWallet,
+  OfflineSignStatus,
+  openExternal,
+  getLiveCells,
+} from 'services/remote'
+import { ReactComponent as HardWalletIcon } from 'widgets/Icons/HardWallet.svg'
+
+import styles from './broadcastTransaction.module.scss'
+
+const BroadcastTransaction = () => {
+  const {
+    app: { loadedTransaction = {} },
+    chain: { networkID },
+    settings: { networks },
+  } = useGlobalState()
+
+  const [wallet, setWallet] = useState<State.Wallet | null>(null)
+  const [isBroadCasting, setIsBroadcasting] = useState(false)
+  const [broadCastedTxHash, setBroadCastedTxHash] = useState('')
+  const [t] = useTranslation()
+  const dispatch = useDispatch()
+  const [errMsg, setErrMsg] = useState('')
+  const isMainnet = isMainnetUtil(networks, networkID)
+
+  const { filePath, json } = loadedTransaction
+
+  const jsonContent = useMemo(() => {
+    return JSON.stringify(json, null, 2)
+  }, [json])
+
+  const signStatus: OfflineSignStatus = json.status
+
+  const isSigned = useMemo(() => signStatus === OfflineSignStatus.Signed, [signStatus])
+
+  const onBack = useGoBack()
+
+  const navigate = useNavigate()
+  const onBroadcast = useCallback(async () => {
+    if (broadCastedTxHash) {
+      openExternal(`${getExplorerUrl(isMainnet)}/transaction/${broadCastedTxHash}`)
+      return
+    }
+
+    setIsBroadcasting(true)
+
+    try {
+      const res = await broadcastTransactionOnly({
+        ...json,
+      })
+      if (isSuccessResponse(res)) {
+        getLiveCells().then(cellRes => {
```

Comment:
I may have deleted a message from @yanguoyu mistakenly, it mentioned that the condition could be `if the broadcasted transaction hash in history of the current wallet`